### ISSUE TITLE
Error message in tfvars is not ending with a dot or question mark

### ISSUE THREAD

**Issue by kirantejaj** [terraform-google-modules/terraform-google-vm#410]
### TL;DR

The module is throwing an error - `validation error must be at least one full English sentence starting with an upper case letter and ending with a period or question mark.`

### Expected behavior

Module should work without any such validation errors.

### Observed behavior

The module is throwing an error - `validation error must be at least one full English sentence starting with an upper case letter and ending with a period or question mark.`

Below error message doesn't have `.` or `?` at the end of sentence.
File - https://github.com/terraform-google-modules/terraform-google-vm/blob/master/modules/instance_template/variables.tf

variable "maintenance_interval" {
  type        = string
  description = "Specifies the frequency of planned maintenance events"
  default     = null
  validation {
    condition     = var.maintenance_interval == null || var.maintenance_interval == "PERIODIC"
    error_message = "var.maintenance_interval must be set to null or \"PERIODIC\""
  }
}

### Terraform Configuration

```hcl
It's failing at the time of initialization itself.
```

### Terraform Version

```sh
0.14.11
```
